### PR TITLE
prettier

### PR DESCRIPTION
**Pull request by wdhdev**

### CODE REVIEW

**Comment by WilliamDavidHarrison** [comment]
@phenax What do you think?

(I spent like an hour doing this since prettier was being stupid lol)

**Review by TheBotlyNoob — DISMISSED** [review]
Quoting @phenax (https://github.com/is-a-dev/register/pull/2664#pullrequestreview-1048149693):
> We can add prettier just for the script files but let's leave the json files alone. Ideally, the subdomain json files must only have commit logs authored by the owner. I used to run a simple formatter in the initial stages of the project but stopped because of this reason.

**Comment by WilliamDavidHarrison** [comment]
It doesn't include JSON files

**Comment by TheBotlyNoob** [comment]
ah, I didn't see the file diff, sorry.

**Review by TheBotlyNoob — DISMISSED** [review]

**Comment by WilliamDavidHarrison** [comment]
> ah, I didn't see the file diff, sorry.

Np

**Comment by WilliamDavidHarrison** [comment]
Oh, we need Phenax to review as well

**Comment by TheBotlyNoob** [comment]
yeah...

**Comment by WilliamDavidHarrison** [comment]
@phenax

**Comment by WilliamDavidHarrison** [comment]
It's because I was modifying script files, I think.

File: scripts/certbot.sh [hunk 1]
```diff
@@ -1,6 +1,5 @@
-#!/usr/bin/env bash
```
Comment (phenax) [inline scripts/certbot.sh:1]: Shebang needs to be here

**Review by phenax — CHANGES_REQUESTED** [review]
Please add a `format` command in package.json scripts so that we have one consistent method for formatting

**Comment by WilliamDavidHarrison** [comment]
> Please add a `format` command in package.json scripts so that we have one consistent method for formatting

I was using prettier online, not a desktop version. Also, prettier isn't installed in `package.json`.

File: scripts/verify-record.sh [hunk 2]
```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env sh
```
Comment (phenax) [inline scripts/verify-record.sh:1]: Needs the shbang

**Review by phenax — CHANGES_REQUESTED** [review]
This and there's a merge conflict in README

**Pull request closed without merging**